utils/iam_dataset.py

```diff
@@ -0,0 +1,298 @@
+import os
+import tarfile
+import urllib
+import sys
+import time
+import glob
+import pickle
+import xml.etree.ElementTree as ET
+import cv2
+import json
+import numpy as np
+import pandas as pd
+import zipfile
+import matplotlib.pyplot as plt
+
+from mxnet.gluon.data import dataset
+from mxnet import nd
+
+class IAMDataset(dataset.ArrayDataset):
+    """ The IAMDataset provides images of handwritten passages written by multiple
+    individuals. The data is available at http://www.fki.inf.unibe.ch
+
+    The passages can be parsed into separate words, lines, or the whole form.
+    The dataset should be separated into writer independent training and testing sets.
+
+    Parameters
+    ----------
+    parse_method: str, Required
+        To select the method of parsing the images of the passage
+        Available options: [form, line, word]
+
+    credentials: (str, str), Default None 
```

Comment:
can you add a `credentials.json.example` at the root of the folder? + docs in the README to copy and rename to `credentials.json` and fill with appropriate credentials from the IAM database.